README.md

```diff
@@ -56,6 +130,8 @@ We are also serving AMP HTML pages at the route `/news/articles/:id.amp` [https:
 - [http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp](http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp)
 - [http://localhost.bbc.com:7080/persian/articles/c4vlle3q337o.amp](http://localhost.bbc.com:7080/persian/articles/c4vlle3q337o.amp).
 
+### Front pages
+
 World Service front pages are served in the format `/:service` where `service` represents a World Service site:
```

Comment:
Suggest that could link to the languages page here https://www.bbc.com/ws/languages to show how languages we will be working with?

Reply:
Above in same file I've linked to https://github.com/bbc/simorgh/wiki/Simorgh-Pages to show progress, I think until we've done a lot of the services it's clearer to link to this link than that one as it would most likely lead people to a page produced by a different application.